test/definitions/super-agent/logs/suse155-super-agent.json

```diff
@@ -0,0 +1,36 @@
+{
+    "global_tags": {
+        "owning_team": "virtuoso",
+        "Environment": "development",
+        "Department": "product",
+        "Product": "virtuoso"
+    },
+    "resources": [
+        {
+            "id": "host1",
+            "provider": "aws",
+            "type": "ec2",
+            "size": "t3.nano",
+            "ami_name": "suse-sles-15-sp5-chost-byos-v????????-hvm-ssd-x86_64"
+        }
+    ],
+    "instrumentations": {
+        "resources": [
+            {
+                "id": "nr_super_agent",
+                "resource_ids": [
+                    "host1"
+                ],
+                "provider": "newrelic",
+                "source_repository": "https://github.com/newrelic/open-install-library.git",
+                "deploy_script_path": "test/deploy/linux/newrelic-cli/install-recipe/roles",
+                "params": {
+                    "newrelic_cli_option": "-n logs-integration",
```

Comment:
The `newrelic_cli_option` param needs to be updated to `"-n super-agent,logs-integration-super-agent",` and "env_var": "NEW_RELIC_CLI_SKIP_CORE=1" param needs to be added.

An example test config with such params here https://github.com/newrelic/open-install-library/blob/main/test/definitions/super-agent/logs/suse154-super-agent-logs.json#L28-L29